pkg/database/migrate.go

```diff
@@ -2,9 +2,10 @@ package database
 
 import (
 	"fmt"
+	"slices"
 
 	log "github.com/sirupsen/logrus"
-	"golang.org/x/exp/slices"
+
 	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
 	"gorm.io/gorm/logger"
```

Comment:
please fix groping.

Reply:
good point — I will fix this everywhere whilst I'm at it

Reply:
done in https://github.com/G-Research/fasttrackml/pull/367/commits/499029f421d742b434361ff8fb85ab19e3374485